cgo/lapack.go

```diff
@@ -576,6 +576,21 @@ func (impl Implementation) Dtrcon(norm lapack.MatrixNorm, uplo blas.Uplo, diag b
 	return rcond[0]
 }
 
+// Dtrtri computes the inverse of a triangular matrix, storing the result in place
+// into a. This is the BLAS level 3 version of the algorithm.
+//
+// Dtrti returns whether the matrix a is singular.
```

Comment:
Returns whether it's singular or whether it's not singular; ok is ambiguous here.